x-pack/legacy/plugins/siem/public/components/events_viewer/index.tsx

```diff
@@ -159,7 +160,7 @@ const StatefulEventsViewerComponent = React.memo<Props>(
           id={id}
           dataProviders={dataProviders!}
           end={end}
-          filters={[...filters, ...defaultFilters]}
+          filters={[...filters, ...pageFilters]}
```

Comment:
++ on the rename to `pageFilters`! 🎉 I think this makes more sense than `defaultFilters` as they're more than just the initial filters. It looks like `defaultFilters` is still being used in `makeMapStateToProps` though (line 214), so we'll want to remove that (no need to replace with pageFilters there, as having in both places will duplicate the filter).